galasa-ui/src/components/PageHeaderMenu.tsx

```diff
@@ -20,53 +19,37 @@ function PageHeaderMenu() {
 
   const handleDeleteCookieApiOperation = async () => {
 
-    try{
+    const response = await fetch('/auth/tokens', { method: 'DELETE' });
 
-      const response = await fetch('/auth/tokens', { method: 'DELETE' });
+    if (response.status === 204) {
 
-      if(response.status === 204){
+      //auto refresh page to render dex login page
+      router.refresh()
```

Comment:
Thinking ahead here: if the user is on a page like `/runs` for example and logs out, but then logs back in again - will they be taken to the homepage or back to the `/runs` page when they log in?

This might need to become a redirect back to the root page (`/`)